inst/shiny-examples/myapp/www/locales/en/dictionary_id.json

```diff
@@ -549,5 +549,6 @@
     "help_ThPb_i2i": "Use the isochron intercept to estimate the initial <sup>208</sup>Pb/<sup>204</sup>Pb-ratio?",
     "PbPb_67": "Measured <sup>206</sup>Pb/<sup>207</sup>Pb-ratios.",
     "PbPb_err_67": "Analytical uncertainty of the <sup>206</sup>Pb/<sup>207</sup>Pb-ratios",
-    "ages_48_0": "The initial <sup>234</sup>U/<sup>238</sup>U-ratio."
+    "ages_48_0": "The initial <sup>234</sup>U/<sup>238</sup>U-ratio.",
+    "translate_button": "<a class=\"translatelink\" href=\"${LINK}\" title=\"Please follow this link to help translate this text\">TRANSLATE</a>"
```

Comment:
Testing GitHub's comment function for the first time...
Why is there a translate_button here?

Reply:
Fallback translations now (where possible) include this button to take the user to the appropriate page in Weblate if they want to contribute a translation. This translate button, of course, needs translating.